Dockerfile

```diff
@@ -12,6 +12,15 @@ RUN git clone http://git.ipxe.org/ipxe.git && \
       sed -i -e "s/#undef.*NET_PROTO_IPV6/#define NET_PROTO_IPV6/g" config/general.h && \
       make bin/undionly.kpxe bin-x86_64-efi/ipxe.efi bin-x86_64-efi/snponly.efi
 
+RUN dd bs=1024 count=2880 if=/dev/zero of=esp.img && \
+      mkfs.msdos -F 12 -n 'ESP_IMAGE' ./esp.img && \
```

Comment:
Why do we pin it to FAT12? I fear we might run into some limit (as it always happens with floppy disks).

I think if we omit `-F` entirely, `mkfs` will figure it automatically. Or we can pin it to FAT32 once and forever.

Reply:
No real reason besides I believe fat12 is truly caseless. I suspect most gear would be fine with fat 16, but a couple of the examples of esp image building were in fat 12. *shrug*

Reply:
Changed in next push.

Reply:
So should have tried building that locally.. .it is looking like the tools failed if it was not fat12. Rebuilding locally with it changed to give it a try.